### PR TITLE
TMA-549 - Ruby SDK Procedure "project.upload_multiple" Does Accept Va…

### PR DESCRIPTION
**Pull request by korczis**
…lid CSV

## Todos
- [ ] Tests
- [ ] Documentation
- [ ] Changelog
- [ ] Build passing

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/49/ : SUCCESS in 4s
- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/888/ : FAILURE in 15s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/35/ : SUCCESS in 19s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/50/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/889/ : SUCCESS in 15s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/36/ : SUCCESS in 12s

**Comment by korczis** [comment]
extended test

File: lib/gooddata/models/model.rb [hunk 1]
```diff
@@ -252,9 +254,31 @@ def upload_multiple_data(data, project_blueprint, options = { :client => GoodDat
           m += "Columns that should be there (manifest) but aren't in uploaded csv: #{manifest_extra}\n" unless manifest_extra.empty?
           m += "Columns that are in csv but shouldn't be there (manifest): #{csv_extra}\n" unless csv_extra.empty?
           m += "Columns in the uploaded csv: #{csv_headers}\n"
-          m += "Columns in the manifest: #{manifest_cols}\n"
+          m += "Columns in the manifest:     #{manifest_cols}\n"
           m += "Original message:\n#{JSON.pretty_generate(js)}\n"
           m += "Manifest used for uploading:\n#{JSON.pretty_generate(manifest)}"
+
+          logs = js['uploads'].flatten.map do |upload|
```
Comment (panjan) [inline lib/gooddata/models/model.rb]: Please shorten this strudel. 😉

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/51/ : SUCCESS in 6s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/828/ : FAILURE in 45m 05s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/601/ : SUCCESS in 19m 45s

**Pull request closed without merging**